### DISCUSSION TITLE
Accessing RSS with redirects active

### DISCUSSION THREAD

**Discussion by natedunn** [wpengine/faustjs#825] [Q&A] [answered]
Hey y'all.

I'm not seeing it right now, but is there a way to safelist certain URLs from redirecting to frontend url? 

I need to have access to the WP RSS feed via `/feed`. Any thoughts or help on this? The only lead I might have was catching the URL before it gets normalized, but that would require me to modified plugin code, which isn't what I want to do.

Thanks in advance!

**Comment by josephfusco** [comment]
Hi @natedunn, thanks for creating this issue!
We've created a ticket ([MERL-312](https://wpengine.atlassian.net/browse/MERL-312)) in our internal backlog to investigate this. We'll write back when we have more details.

**Comment by josephfusco** [comment, marked as answer]
@natedunn Version 0.7.11 of [FaustWP](https://wordpress.org/plugins/faustwp/) is now available which prevents WordPress feed URLs from being redirected to the front-end!
